docs/source/readme_link.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../../README.md
+```
```

Comment:
Is it needed? I would expect you can just directly... 
```
.. include:: ../../README.md
   :parser: myst_parser.sphinx_
```
... in index.rst. (Also applies to CONTRIBUTING.)

Reply:
Thanks for that.
It worked for README (so I removed the unnecessary extra file), but not for CONTRIBUTING. I guess because the latter is specified in the TOC (not simply included in an existing file). I could be missing sth, but having an extra "link" file in `docs/source` is the only solution that works for me so far

Reply:
Ah, I see the problem. [I've just checked out `add-docs` and had a play]
Your way is the most explicit that I've come up with, and that works. And therefore the best.